pettingzoo/utils/wrappers/base_parallel.py

```diff
@@ -62,32 +46,6 @@ def unwrapped(self) -> ParallelEnv:
     def state(self) -> np.ndarray:
         return self.env.state()
 
-    @property
```

Comment:
And what about removing this

Reply:
Nothing related to this is implemented in this layer. These should be defined in the environment, or the wrapper modifying the obs space IMO?

Reply:
Makes sense, just saw that it was still there in the AEC base wrapper I think